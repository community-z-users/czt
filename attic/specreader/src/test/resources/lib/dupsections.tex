\begin{zsection}
\SECTION dupsections
\end{zsection}
\begin{zsection}
\SECTION dupsections
\end{zsection}
