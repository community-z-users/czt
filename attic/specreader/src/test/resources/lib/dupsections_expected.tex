net.sourceforge.czt.specreader.DupSectionNameException: Multiple sections with same name dupsections in /lib/dupsections.tex and /lib/dupsections.tex